osquery/sql/sqlite_util.cpp

```diff
@@ -361,11 +366,45 @@ SQLiteDBInstance::~SQLiteDBInstance() {
 SQLiteDBManager::SQLiteDBManager() : db_(nullptr) {
   sqlite3_soft_heap_limit64(1);
   setDisabledTables(Flag::getValue("disable_tables"));
+  setEnabledTables(Flag::getValue("enable_tables"));
 }
 
 bool SQLiteDBManager::isDisabled(const std::string& table_name) {
-  const auto& element = instance().disabled_tables_.find(table_name);
-  return (element != instance().disabled_tables_.end());
+  bool disabled_set = !(instance().disabled_tables_.find("Not Specified") !=
```

Comment:
Instead of checking for this canary value you might be able to use `Flag::isDefault("disable_tables")`. This should check more than if the value is blank/not-blank but rather if the value was set via a flagfile or the CLI.

Reply:
Yes, thanks for the hint